packages/donor/src/components/basic/ZMScreen/ZMScreen.tsx

```diff
@@ -6,20 +6,20 @@ interface ExtendedSignupScreenProps {
   children: React.ReactNode;
   className?: string;
   hasBackButton?: boolean;
-  hasProfileButton?: boolean;
+  hasBurgerMenu?: boolean;
   title: string;
 }
 
 export default function ZMScreen({
   children,
   className = "",
   hasBackButton,
-  hasProfileButton,
+  hasBurgerMenu,
   title,
 }: ExtendedSignupScreenProps) {
   return (
     <>
-      <AppHeader {...{ hasBackButton, hasProfileButton, title }} />
+      <AppHeader {...{ hasBurgerMenu, hasBackButton, title }} />
```

Comment:
Should we somehow enforce we don't get both back button and burger menu?

Reply:
Don't think we should as in theory there is no reason why other pages should not be able to show both.